scanner/scanner.go

```diff
@@ -1,18 +1,28 @@
 package scanner
 
 import (
+	"sync/atomic"
+
 	"github.com/brimsec/zq/filter"
 	"github.com/brimsec/zq/pkg/nano"
 	"github.com/brimsec/zq/proc"
 	"github.com/brimsec/zq/zbuf"
 	"github.com/brimsec/zq/zng"
+	"github.com/brimsec/zq/zqd/api"
 )
 
 // Scanner implements the proc.Proc interface.
 type Scanner struct {
 	reader zbuf.Reader
 	filter filter.Filter
 	span   nano.Span
+	stats  struct {
+		currentTs      int64
```

Comment:
Doesn't look like this field is being used.

Reply:
good catch